packages/slice-machine/lib/utils/version.ts

```diff
@@ -0,0 +1,49 @@
+import path from "path";
```

Comment:
I think we should avoid creating file in lib if the functions defined inside are used at only one place in the server. Create this file inside the services folder in the server folder.